src/Entity/EntityTrait.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Acquia\LiftClient\Entity;
+
+use Symfony\Component\Serializer\Encoder\JsonEncoder;
+use Symfony\Component\Serializer\Normalizer\CustomNormalizer;
+use Symfony\Component\Serializer\Serializer;
```

Comment:
Not really, these are really powerful classes. I prefer to keep them until it becomes an issue. For Drupal 7 there is no restriction in actually committing them to the repo btw. So we might be able to do just that as well.